dvc.lock

```diff
@@ -49,12 +49,13 @@ stages:
     deps:
```

Comment:
Note that nothing in the `00-ingest` stage has changed, only the downstream hashes of that same data (recorded as inputs for subsequent stages).